crates/rpc/src/v02/method/get_class_hash_at.rs

```diff
@@ -207,14 +213,14 @@ mod tests {
             contract_address: address,
         };
         let result = get_class_hash_at(context.clone(), input).await.unwrap();
-        assert_eq!(result, expected);
+        assert_eq!(result.0, expected);
 
         let input = GetClassHashAtInput {
             block_id: StarknetBlockNumber::new_or_panic(2).into(),
             contract_address: address,
         };
         let result = get_class_hash_at(context, input).await.unwrap();
-        assert_eq!(result, expected);
+        assert_eq!(result.0, expected);
```

Comment:
Question: just wondering if adding `Eq` + `PartialEq` or `Ordering` against `Felt` directly makes any sense? That would avoid clumsy `.0` everywhere.

Reply:
It is something to consider - it would be nice to enable it for `#[cfg(test)]` only. This preserves type safety in prod (for whatever reason this might happen..) but allows for much nicer tests.

I think in general our `Felt` and its newtype wrappers could use some TLC (tender love & care), especially with regards to simplify tests.